calico-policies/private-network-isolation/au-syd/allow-private-services-pods.yaml

```diff
@@ -51,6 +51,11 @@ spec:
       - 166.9.52.5/32
       - 166.9.56.3/32
       - 166.9.56.20/32
+      # CIDR Blocks added on 7/26/2023
+      -  10.139.31.0/26
+      -  10.195.95.64/26
+      -  10.63.207.128/26
+      # End CIDR Blocks
```

Comment:
If it turns out they are needed, can you please remove the extra space between the `-` and the `1` so it matches the existing ones in the list?

Reply:
@bradbehele Fixed the spaces. Please can you approve and merge. Also let me know if you want me to cleanup the IP address list so that I can remove the old ones that are no longer in use for all regions and keep the list up-to-date

Reply:
Thanks for fixing up the spaces.  Before merging I would still like to understand how customer cluster workers are able to send traffic to LogDNA via 10.x.x.x destination IPs? I thought the only approved way to send traffic to services like LogDNA over the private network is to use CSEs?  To put it another way, I don't think these 10.x.x.x IPs are useful in this policy because I don't think workers can route traffic to them (if I'm wrong please let me know how, thanks).

Reply:
As far as cleaning up old IPs, if you know that no other clusters (even older existing clusters) will be using the old IPs, then yes, please remove them from the policies, either in this PR or a follow on PR.

Reply:
@bradbehle I have removed all 10.x CIDR blocks per your suggestion above. Please review and merge